src/views/blocks/tickets/submit-button.php

```diff
@@ -53,5 +53,5 @@
 	<?php endif; ?>
 	<?php tribe_disabled( true ); ?>
 >
-	<?php echo esc_html_x( 'Get Tickets', 'Add tickets to cart.', 'event-tickets' ); ?>
+	<?php echo esc_html( sprintf( _x( 'Get %s', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets') ) ); ?>
```

Comment:
the context in `_x()` should be kept:

```suggestion
	<?php echo esc_html( sprintf( _x( 'Get %s', 'Add tickets to cart.', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets') ) ); ?>
```